static/app/components/workflowEngine/ui/container.tsx

```diff
@@ -0,0 +1,15 @@
+import styled from '@emotion/styled';
+
+import {space} from 'sentry/styles/space';
+
+export const Container = styled('div')`
+  display: flex;
+  flex-direction: column;
+  gap: ${space(2)};
+  justify-content: flex-start;
+  background-color: ${p => p.theme.backgroundSecondary};
+  border: 1px solid ${p => p.theme.translucentBorder};
+  border-radius: ${p => p.theme.borderRadius};
+  padding: ${space(1.5)};
+  width: fit-content;
```

Comment:
Can we avoid setting a specific `width`? Typically these kinds of components can just grow to the size of their parent with `flex-grow: 1`.